python_modules/libraries/dagster-aws/dagster_aws/ecs/launcher.py

```diff
@@ -107,19 +107,30 @@ def launch_run(self, context: LaunchRunContext) -> None:
         # Run a task using the same network configuration as this processes's
         # task.
 
-        response = self.ecs.run_task(
-            taskDefinition=task_definition,
-            cluster=metadata.cluster,
-            overrides={"containerOverrides": [{"name": self.container_name, "command": command}]},
-            networkConfiguration={
+        ecs_run_task_args = {
+            "taskDefinition": task_definition,
+            "cluster": metadata.cluster,
+            "overrides": {
+                "containerOverrides": [{"name": self.container_name, "command": command}]
+            },
+            "networkConfiguration": {
                 "awsvpcConfiguration": {
                     "subnets": metadata.subnets,
                     "assignPublicIp": "ENABLED",
                     "securityGroups": metadata.security_groups,
                 }
             },
-            launchType="FARGATE",
-        )
+            "launchType": "FARGATE",
+        }
+
+        try:
+            response = self.ecs.run_task(
+                **ecs_run_task_args,
+                propagateTags="TASK_DEFINITION",
```

Comment:
Instead of always propagating tags from the task definition, I think we should make this configurable in `dagster.yaml`. Similar to how we added the ability to configure secrets recently: https://github.com/dagster-io/dagster/pull/5614/commits/c1fe46c29913ce38a17d20c141217572136e375c

That way, users can choose whether to set this to `TASK_DEFINITION` or leave it blank (the default). Perhaps the config should actually be a boolean - if `propogate_tags == True`, we'll include `propagateTags="TASK_DEFINITION"`; otherwise we won't.